app/models/account.rb

```diff
@@ -8,12 +8,10 @@ class Account < ApplicationRecord
 
   # an account has user optionally
   has_one :user
-
   has_many :automatic_topup_configs, class_name: 'AccountAutomaticTopupConfig', foreign_key: :to_account_id
-
   has_many :objectives
-
   belongs_to :parent, class_name: 'Account', optional: true
+  has_one_attached :avatar
```

Comment:
Please configure a dynamic storage - in dev it's a file system. In prod it's AWS. Also, prepare the code for AWS.